devenv/doctor.py

```diff
@@ -173,12 +170,8 @@ def main(context: Dict[str, str], argv: Sequence[str] | None = None) -> int:
 
     match_tags: set[str] = set(args.tag if args.tag else ())
 
-    repo = context["repo"]
-    if repo not in {"sentry", "getsentry", "devenv"}:
```

Comment:
repo not supported yet is outdated, i've been getting rid of it

last one is removed in https://github.com/getsentry/devenv/pull/77